gen-ai/sample-claude-multimodal.json

```diff
@@ -0,0 +1,108 @@
+{
+  "variable": [
+    {
+      "value": "https://api.anthropic.com/v1/messages",
+      "key": "api.url"
+    },
+    {
+      "value": "claude-3-opus-20240229",
+      "key": "llm"
+    },
+    {
+      "value": "<your-prompt-here>",
+      "current": "",
+      "key": "textInput"
+    },
+    {
+      "value": "<image-in-base64>",
+      "current": "",
+      "key": "imgBase64"
+    },
+    {
+      "value": "0",
+      "current": "",
+      "key": "temp"
```

Comment:
you're sure temp is working here?  For me it was throwing some error converting what looked like a string to a number...

Reply:
It works for me. The API Test Editor highlights it as an error even though it isn't: https://mabl.atlassian.net/issues/MABL-13595